static/styles/settings.css

```diff
@@ -227,6 +227,7 @@ h3,
 
 .table-condensed td {
     vertical-align: middle;
+    padding: 8px 5px;
```

Comment:
I am not sure whether adding padding to all the cells is right solution here and that too of all the tables in the app.

I think one solution can be to instead show the tooltips for this particular table at bottom, but we can have feedback from others too.